rozenborg/test-simulations-mgmt-repo
# PR for all completed ouputs

#### What does this PR do?
It shows;
-  Setting up front-end for `writing-html-js-css` 
- InvertedIndex class for `solving-the-problem` output
- Test specs for `writing-tests` output
- Link to a successful TravisCI build for`setting-up-continuous-integration` output
- Link to medium post for `reflecting-on-the-learning-experience` output
- Link to herokuapp pages for `deployment` output
- Link to release notes for `delivering-your-product` output
- Two design patterns for `using-design-patterns` output

#### Description of Task to be completed?
- A Pull Request showing your implementation of the solution (e.g. InvertedIndex class)
- A link to a successful TravisCI build of your `development` branch (it should include test runs)
- A folder containing the template for the proposed front-end of your project
- A Pull Request showing all tests required for your project using Mocha/Jasmine with any assertion library of choice (Chai.JS, Should.JS, Expect.JS...)
- A post reflecting on your learning experience while producing these outputs
- Live URLs to a `review` app, `staging` & `production` apps
- `Release notes` for your final product
- A Pull Request showing two Design Patterns (suggestions: Decorator, Factory, Observer, Prototype...)

#### How should this be manually tested?

- For `writing-html-js-css` output: Check Screenshots below:
- For `solving-the-problem` output: View [Solving the problem](https://github.com/andela/test-simulations-mgmt-repo/tree/bayo-izanagi-oduyile/06-building-a-program-start-to-finish/solving-the-problem/output) output folder
- For `writing-tests` output: Visit [Test output folder](https://github.com/andela/test-simulations-mgmt-repo/tree/bayo-izanagi-oduyile/06-building-a-program-start-to-finish/writing-tests/output/)
- For`setting-up-continuous-integration` output: [Travis on Development](https://travis-ci.org/andela-oduyile/Inverted-Index-App)
- For `reflecting-on-the-learning-experience` output: [Medium post](https://medium.com/@oluwatomi.duyile/the-journey-to-become-a-world-class-developer-27d905986899#.7v9ffg1ch")
- For `deployment` output: Click the links below
 1. [Review App](https://indexit-pr-10.herokuapp.com/) 2. [Staging App](https://indexit-staging.herokuapp.com/) 3. [Production App](https://indexit.herokuapp.com/)

- For `delivering-your-product` output: [Release Document](https://docs.google.com/document/d/1LCc83vH6kCfGZIB1vxNpqT_51nWHoVzJB6L2kQdicAs/edit)
- For `using-design-patterns` output: [Test output folder](https://github.com/andela/test-simulations-mgmt-repo/tree/bayo-izanagi-oduyile/06-building-a-program-start-to-finish/using-design-patterns/output)

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)

<img width="1219" alt="screen shot 2017-03-16 at 9 32 18 pm" src="https://cloud.githubusercontent.com/assets/25608336/24043372/9bfff438-0b16-11e7-8372-e2d799d895a5.png">
<img width="1214" alt="screen shot 2017-03-16 at 8 00 29 pm" src="https://cloud.githubusercontent.com/assets/25608336/24043370/9bff5258-0b16-11e7-8f09-7dd22c541828.png">
<img width="1234" alt="screen shot 2017-03-17 at 1 42 06 am" src="https://cloud.githubusercontent.com/assets/25608336/24043373/9c00eadc-0b16-11e7-9115-b2d670dc43d4.png">
<img width="1220" alt="screen shot 2017-03-17 at 1 43 23 am" src="https://cloud.githubusercontent.com/assets/25608336/24043374/9c026df8-0b16-11e7-8f6b-7ddbc54a50d8.png">
<img width="1258" alt="screen shot 2017-03-17 at 1 45 44 am" src="https://cloud.githubusercontent.com/assets/25608336/24043371/9bffd1c4-0b16-11e7-9925-7a9519864f99.png">
<img width="1234" alt="screen shot 2017-03-17 at 4 28 27 am" src="https://cloud.githubusercontent.com/assets/25608336/24043375/9c249e46-0b16-11e7-85d2-3365848fce9e.png">
